application-xpoll-api/src/main/resources/vote.xsd

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ *
+ * See the NOTICE file distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU Lesser General Public License as
+ * published by the Free Software Foundation; either version 2.1 of
+ * the License, or (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this software; if not, write to the Free
+ * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
+ * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
+ *
+-->
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" targetNamespace="http://www.xwiki.com/xpoll/model/jaxb"
+  xmlns:xpoll="http://www.xwiki.com/xpoll/model/jaxb" elementFormDefault="qualified">
+
+  <xs:element name="xpoll" type="xpoll:Vote"/>
+
+  <xs:complexType name="Vote">
+    <xs:sequence>
+      <xs:element name="votedProposals" maxOccurs="unbounded" type="xs:string"/>
```

Comment:
```suggestion
      <xs:element name="proposal" maxOccurs="unbounded" type="xs:string"/>
```

The model generated by JAXB makes it plural, but in the XML it should be multiple ``<proposal>`` tags:

```
<vote>
  <proposal>One</proposal>
  <proposal>Three</proposal>
</vote>
```

I'd drop the "voted" prefix, because we're inside the vote element so it's clear that these are the voted proposals.